.gitignore

```diff
@@ -8,3 +8,6 @@ h2spec
 # Files generated by honggfuzz
 hfuzz_target
 hfuzz_workspace
+
+# Editor-specific settings
+.vscode
```

Comment:
This should be moved to a system global gitignore instead of in each repo.

Reply:
Huh. Honestly never though about doing that. I'll remove that one then.